### ISSUE TITLE
BioPhi pinned to an old version of ANARCI

### ISSUE THREAD

**Issue by TheUG93** [Merck/BioPhi#67]
I am interested in using BioPhi with the updated version of ANARCI. This is currently not possible as BioPhi is pinned to the 2024.04.23 version. Happy to put in a pull request if there is interest.

**Comment by prihoda** [comment]
At the time it was due to this bug, not sure if it has been fixed yet: https://github.com/oxpig/ANARCI/issues/17

However, you can update ANARCI after installing biophi and it should work - just no guarantees :)

**Comment by TheUG93** [comment]
Thanks! Seems like the bug still exists in the latest conda version. I did notice that passing "human" to the use_species flag seems to fix the example in the linked issue (mouse seems to work as well). The use_species flag was updated going from the old version to new version so its possible that the default behavior was updated. I will have to do some digging around to confirm. 

I did also run the example using ANARCI installed directly from the repository and it seems that the bug has been fixed. The default has been updated to use human and mouse as the allowed species. So, a new ANARCI conda package based on the current build should fix the issue. 

Closing this ticket as there is already a relevant open ticket ([original ticket](https://github.com/Merck/BioPhi/issues/9))
